### PR TITLE
Sharing Language Keys Sample

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

File: liferay-workspace/modules/blade.language.web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<p>
+	<b><liferay-ui:message key="blade_language_web_LanguageWebPortlet.caption" /></b>
```
Comment (gamerson) [inline liferay-workspace/modules/blade.language.web/src/main/resources/META-INF/resources/view.jsp]: Lets show 3 captions here.

1) one caption load from blade.language module solely
2) one caption that is using a key that is defined in both language and langauge.web to show that language.web overrides it
3) one caption that only exists in language.web

**Review by gamerson — CHANGES_REQUESTED** [review]

**Comment by lawrence-lee** [comment]
@gamerson tests passed!

**Comment by gamerson** [comment]
@lawrence-lee Sweet!

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-blade-samples/pull/216. [See changes here.](https://github.com/gamerson/liferay-blade-samples/compare/c54f285~1...c54f285) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/gamerson/liferay-blade-samples/pull/493. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**